docs/preview_conversion_analyzer.md

```diff
@@ -0,0 +1,172 @@
+[More on WinUI](https://microsoft.github.io/microsoft-ui-xaml/), [Docs](https://docs.microsoft.com/en-us/windows/apps/winui/)
+- WinUI3 is the next version of WinUI. It runs on the native Windows 10 UI platform and supports both Windows Desktop and UWP apps. WinUI3 ships as a NuGet package.
+[More on WinUI3](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/)
+
+Converting an existing WinUI C# App to WinUI3 requires some changes to the C# source code. Most notably namespace changes from Windows.UI.* to Microsoft.UI.*
+
+This porting assistance is provided in the form of Code Analyzers and Code Fixes. 
+- A Code Analyzer extends the Visual Studio experience and provides on the fly code inspections for C#. For most inspections these Code Analyzers provide light bulb quick fixes that can be applied to users code in the form of a Code Fix. 
+- A Code Fix consumes the diagnostics created by the Code Analyzer and modifies the C# file in-line. Users may choose to apply Code Fixes one at a time or resolve all instances of a Code Fix across a File, Project, or Solution.
+
+# What The Conversion Analyzers Do:
+Analyze/Convert C# files and apply the following changes:
+
+## Changes Common to All Projects:
+
+### Namespace Analyzer/Codefix
+
+- Updates Type Namespaces from `Windows.*` to `Microsoft.*`
+- Types moving from `Windows` to `Microsoft`:
+    - `Windows.UI.Xaml`
+    - `Windows.UI.Colors`
+    - `Windows.UI.ColorHelper` 
+    - `Windows.UI.Composition`
+    - `Windows.UI.Input`
+    - `Windows.UI.Text`
+    - `Windows.System.DispatcherQueue*` 
+
+### EventArgs Analyzer/Codefix
+
+- Converts `App.OnLaunched` Method
+    - Two updates need to be made to the `App.OnLaunched` method when converting to WinUI3:
+    1. Target `Microsoft.UI.Xaml.LaunchactivatedEvenArgs` as the method parameter type
+    2. Instances of the parameter name in the `App.OnLaunched` method body must invoke `UWPLaunchActivatedEventArgs`
+
+## Changes Specific to UWP (.NET Native) Projects:
+
+### UWP Projection Analyzer/Codefix
+
+- Some types are moving in UWP and projects need to target the new projected types.
+- .Net Projections moving to `Microsoft.UI.Xaml` :
+    - `System.ComponentModel.INotifyPropertyChanged` -> `Microsoft.UI.Xaml.Data.INotifyPropertyChanged`
+    - `System.ComponentModel.PropertyChangedEventArgs`-> `Microsoft.UI.Xaml.Data.PropertyChangedEventArgs`
+    - `System.Windows.Input.ICommand` -> `Microsoft.UI.Xaml.Input.ICommand`
+
+### ObservableCollection Analyzer/Codefix
+
+- `ObservableCollection<T>` is being removed and users will have to provide their own implementation targeting `INotifyCollectionChanged`.
+- If the analyzer cannot find an implementation it will provide its own helper class.
+    - `System.Collections.ObjectModel.ObservableCollection` -> `Microsoft.UI.Xaml.Interop.INotifyCollectionChanged`
+        
+### UWP Struct Analyzer/Codefix
+
+- UWP Projects cannot use struct constructors. 
```

Comment:
I think this detail belongs with the before/after.  Coupled with the bullet list that has a link to the more detailed section, I think the whole doc will flow better.  

Then you can make your sections look like this:

### UWP Struct Analyzer/CodeFix

*Details here*

#### Example Before

```
Code here
```
#### Example After

```
Code here
```

Reply:
Modified the sections

Reply:
Restructured sections to match